MapboxCoreNavigation/RouteController.swift

```diff
@@ -142,10 +166,33 @@ open class RouteController: NSObject {
         super.init()
         
         self.locationManager.delegate = self
+        
+        self.resetSession()
+        
+        self.resumeNotifications()
+        
+        if let accessToken = MGLAccountManager.accessToken() {
```

Comment:
Is this the only reason Core Navigation depends on the map SDK? We should instead look directly at `MGLMapboxAccessToken` in Info.plist. There’s a chance the developer set the map access token using `MGLAccountManager.setAccessToken(_:)`, which means `MGLMapboxAccessToken` is `nil`. Regardless, I think it’s more important for directions-related telemetry to use the same access token as MapboxDirections.swift (`Directions.accessToken`), which isn’t necessarily the same access token that MGLAccountManager sees.